backend/docs/auth.md

```diff
@@ -0,0 +1,53 @@
+# <u>Backend Documentation</u>
+</br>
+
+### Auth module
+This module is responsible for all the activities related to authentication.
+
+### API description
+-  [ <b>POST</b> ] : <code> <b> /auth/login </b> </code>
+    - Login for existing users
+    - request body : 
+        | Parameter   | Description|
+        | ----------- | -----------|
+        | email       | string , required|
+        | password    | string , required|
+
+    - ![diagram](images/loginroute.jpg)
+    <br>
+##### The auth middleware uses JWT method for verification.
+<br>
+
+![diagram](images/jwt.jpg)
+<br>
+JWT consists of three parts headers , payload and signature.
```

Comment:
Readability can be improved over here
Please follow github markdown rules, Yours arn't working i guess.
Whenever you need to use a comma the gramattical syntax is like
`Hi, this is how we use comma` i.e after comma we leave a space and not before. Would be nice if you can refactor